imports/ui/validators/TimeDistubtionChart.jsx.js

```diff
@@ -182,13 +182,13 @@ export default class TimeDistubtionChart extends Component{
         let data = this.populateChartData();
         return [
             <Card key='timeilne'>
-                <CardHeader>History Missed Blocks</CardHeader>
+                <CardHeader>History Missed <span className='capitalize'>{this.props.type}</span></CardHeader>
                 <CardBody>
                     <PChart {...this.populateTimelineChart(data.timeline)} />
                 </CardBody>
             </Card>,
             <Card key='breakdown'>
-                <CardHeader>Missed Blocks By Time of Day</CardHeader>
+                <CardHeader>Missed <span className='capitalize'>{this.props.type}</span> By Time of Day</CardHeader>
```

Comment:
I prefer using Bootstrap text utility classes for text transformation and this can get rid of creating new classes in the CSS file.

https://getbootstrap.com/docs/4.0/utilities/text/#text-transform

I think after this update, it's good to merge. The paging issue for the large number of missing only affect those high misses validators and it's not a very important case to solve now.

Thanks!